src/main/java/com/fasterxml/jackson/databind/ext/NioPathSerializer.java

```diff
@@ -0,0 +1,24 @@
+
+package com.fasterxml.jackson.databind.ext;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+/**
+ *
+ */
+public class NioPathSerializer extends JsonSerializer<Path> {
+    @Override
+    public void serialize(Path value, JsonGenerator gen, SerializerProvider serializers) throws IOException {
+        if (value == null) {
```

Comment:
Nulls are not passed to either serializers or deserializers, so this can be simplified.